### PR TITLE
Create 2021 Week 3 announcement.md

### PR DESCRIPTION
**Pull request by cokiengchiara**
Opening PR to start announcement w recently released Prometheus Operator https://github.com/giantswarm/roadmap/issues/227
General Checklist: https://intranet.giantswarm.io/docs/product/releases/announcement-checklist/

- [x] Add announcement of Renaming "control plane" to "management cluster" and "tenant cluster" to "workload cluster" towards https://github.com/giantswarm/giantswarm/issues/14961 cc @giantswarm/chapter-product 
- [x] Add Aqua 5.3.0 https://github.com/giantswarm/aqua-app/blob/master/CHANGELOG.md#530---2021-01-12, towards https://github.com/giantswarm/roadmap/issues/227
- [x] Add prometheus-operator https://docs.giantswarm.io/changes/managed-apps/
- [x] Add cert-manager ^^
- [x] Add EFK ^^
- [x] Add Jaeger https://docs.giantswarm.io/changes/playground-apps/
- [x] Update week number to this week (22. Jan 2021)

### CODE REVIEW

File: announcements/2021-Week-02/announcement.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# :zap: This Week in Provider-Independent Announcements (January 15, 2021):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+[Prometheus Operator v0.5.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.5.0/) upgrades to the chart with Prometheus v2.22.1.
```
Comment (cokiengchiara) [inline announcements/2021-Week-02/announcement.md:7]: @TheoBrigitte @glitchcrab Can you suggest a change if there are other customer-facing changes or improvements please?

Comment (TheoBrigitte) [reply announcements/2021-Week-02/announcement.md:7]: Notables changes are:

* prometheus-operator v0.42.1 -> v0.43.2: introduces a new AlertmanagerConfig CRD. https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.43.0
* prometheus v2.21.0 -> v2.22.1 : the experimental gRPC API v2 has been
removed. https://github.com/prometheus/prometheus/releases/tag/v2.22.0

**Comment by marians** [comment]
Committed a paragraph regarding terminology.

File: announcements/2021-Week-02/announcement.md (hunk 1 above)
Comment (MartaD) [reply announcements/2021-Week-02/announcement.md:7]: is this already included in the announcement and if not can you add it?

**Comment by MartaD** [comment]
@cokiengchiara there's a lot of unticked boxes at the top, will you be adding these apps to the announcement?

**Comment by MartaD** [comment]
@giantswarm/team-celestial is this worth mentioning in the weekly announcement for customers:
https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v1.20.0/ ?

**Comment by MartaD** [comment]
@giantswarm/team-firecracker is any of the changes in Management API changes in January worth mentioning to customers: https://docs.giantswarm.io/changes/management-api/

File: announcements/2021-Week-03/announcement.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Managed Apps
+
+[Prometheus Operator v0.5.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.5.0/) upgrades to the chart with Prometheus v2.22.1.
+
+## Web UI
+
+## kubectl gs
+
+- Added support for getting nodepools and removed mention of the unexistent ‘create cluster’ command.
```
Comment (MartaD) [inline announcements/2021-Week-03/announcement.md:19]: @whites11 is this for Azure only or AWS also?

Comment (axbarsan) [reply announcements/2021-Week-03/announcement.md:19]: Both Azure and AWS

Comment (MartaD) [reply announcements/2021-Week-03/announcement.md:19]: Thanks!

File: announcements/2021-Week-02/announcement.md (hunk 1 above)
Comment (cokiengchiara) [reply announcements/2021-Week-02/announcement.md]: I will add it

**Comment by glitchcrab** [comment]
no complaints here

File: announcements/2021-Week-03/announcement.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- The API formerly called *Control Plane Kubernetes API* is now called *Management API*.
+- What we called *tenant cluster* is now called a *workload cluster*.
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+[Prometheus Operator v0.5.2](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.5.2/) introduces a new AlertmanagerConfig CRD and upgrades to Prometheus v2.22.1.
+
+[EFK Stack v0.4.1](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.4.1/) fixes an issue in v0.4.1, which upgrades to Elasticsearch and Kibana v7.9.1.
```
Comment (webwurst) [inline announcements/2021-Week-03/announcement.md:17]:
```suggestion
[EFK Stack v0.4.1](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.4.1/) came quickly after v0.4.0 to fix a small oversight. With v0.4.x Elasticsearch and Kibana are running in version v7.10.0.
```

File: announcements/2021-Week-03/announcement.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- The term *management cluster* replaces the term *control plane*. The Kubernetes project uses the term control plane when dealing with the master nodes (as they were called formerly) of a cluster, so we are removing an ambiguity that confused people.
+- The API formerly called *Control Plane Kubernetes API* is now called *Management API*.
+- What we called *tenant cluster* is now called a *workload cluster*.
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+[Loki v0.1.0-alpha](https://github.com/giantswarm/loki-app/blob/master/CHANGELOG.md#010-alpha---2021-01-21), our preferred app for log management is now available. For more info, see [blog post](https://www.giantswarm.io/blog/how-the-cloud-native-stack-helps-writing-minimal-microservices/).
```
Comment (cokiengchiara) [inline announcements/2021-Week-03/announcement.md:13]:
```suggestion
[Loki v0.1.0-alpha](https://github.com/giantswarm/loki-app/blob/master/CHANGELOG.md#010-alpha---2021-01-21), our preferred app for log management is now available. Promtail, the logging client, is coming soon. Interested in Loki? We'd like to hear from you.
```

**Comment by cokiengchiara** [comment]
All good from Halo's side! @MartaD

**Review by MartaD — APPROVED** [review]
LGTM

**Pull request merged**